### ISSUE TITLE
Doesn't work with pipewire-jack

### ISSUE THREAD

**Issue by TheK0tYaRa** [SpotlightKid/python-rtmidi#106]
`/usr/bin/ld: cannot find -ljack`

**Comment by SpotlightKid** [comment]
What does `pkg-config --libs jack` return?

**Comment by TheK0tYaRa** [comment]
`-L/usr/lib/pipewire-0.3/jack/ -ljack`

**Comment by mbollmann** [comment]
As a hotfix (if anyone else stumbles on this, like I did), download the source distribution from PyPi and edit this part at the end of `setup.py`:

```python

# define _rtmidi Extension
extensions = [
    Extension(
        PKG_DIR + "._rtmidi",
        sources=sources,
        language="c++",
        define_macros=define_macros,
        include_dirs=include_dirs,
        libraries=libraries,
        library_dirs=["/usr/lib64/pipewire-0.3/jack/"],
        extra_compile_args=extra_compile_args,
        extra_link_args=extra_link_args
    )
]
```

...where the dir after library_dirs is whatever the directory returned by `pkg-config --libs jack` is.

Then `make dist && pip install dist/python_rtmidi-1.4.9-cp39-cp39-linux_x86_64.whl` worked for me.

**Comment by SpotlightKid** [comment]
I plan to take the time in the next two weeks to fix outstanding issues like these and do a new release, so thanks for your input.

**Comment by brunoenten** [comment]
The build system was completely overhauled, and binary wheels are available for recent python versions.
Does it solve this issue?

**Comment by brunoenten** [comment]
That's it, the 1.5.0 release with binary wheels for recent cpython has been released on pypi!
https://pypi.org/manage/project/python-rtmidi/release/1.5.0/

Please let me know if `pip install python-rtmidi` now works for you and if it fixes your issues.

**Comment by mbollmann** [comment]
Thank you @brunoenten! Installing and importing the library works like a charm on my current system, but both my hardware and OS have changed since I last tried this, and I believe I'm running it with ALSA rather than JACK now. So maybe some JACK user can confirm?

**Comment by SpotlightKid** [comment]
@TheK0tYaRa Can you confirm whether this is solved?

**Comment by SpotlightKid** [comment]
Assumed fixed with release 1.5.0.
